src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,111 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private String filePath;
+    private PrintWriter storeInFile;
+    private boolean filePresent = false;
```

Comment:
Perhaps this could be renamed as isFilePresent following the convension for naming bool variables?